src/java.compiler/share/classes/javax/annotation/processing/Processor.java

```diff
@@ -90,35 +90,35 @@
  * configuration mechanisms, such as command line options; for
  * details, refer to the particular tool's documentation.  Which
  * processors the tool asks to {@linkplain #process run} is a function
- * of the types of the annotations <em>{@linkplain AnnotatedConstruct present}</em>
+ * of the interfaces of the annotations <em>{@linkplain AnnotatedConstruct present}</em>
  * on the {@linkplain
  * RoundEnvironment#getRootElements root elements}, what {@linkplain
- * #getSupportedAnnotationTypes annotation types a processor
+ * #getSupportedAnnotationTypes annotation interfaces a processor
  * supports}, and whether or not a processor {@linkplain #process
- * claims the annotation types it processes}.  A processor will be asked to
- * process a subset of the annotation types it supports, possibly an
+ * claims the annotation interfaces it processes}.  A processor will be asked to
+ * process a subset of the annotation interfaces it supports, possibly an
  * empty set.
  *
- * For a given round, the tool computes the set of annotation types
+ * For a given round, the tool computes the set of annotation interfaces
  * that are present on the elements enclosed within the root elements.
- * If there is at least one annotation type present, then as
- * processors claim annotation types, they are removed from the set of
- * unmatched annotation types.  When the set is empty or no more
+ * If there is at least one annotation interface present, then as
+ * processors claim annotation interfaces, they are removed from the set of
+ * unmatched annotation interfaces.  When the set is empty or no more
  * processors are available, the round has run to completion.  If
- * there are no annotation types present, annotation processing still
+ * there are no annotation interfaces present, annotation processing still
  * occurs but only <i>universal processors</i> which support
- * processing all annotation types, {@code "*"}, can claim the (empty)
- * set of annotation types.
+ * processing all annotation interfaces, {@code "*"}, can claim the (empty)
+ * set of annotation interfaces.
  *
- * <p>An annotation type is considered present if there is at least
- * one annotation of that type present on an element enclosed within
+ * <p>An annotation interface is considered present if there is at least
+ * one annotation of that interface present on an element enclosed within
  * the root elements of a round. For this purpose, a type parameter is
  * considered to be enclosed by its {@linkplain
  * TypeParameterElement#getGenericElement generic
  * element}.
 
  * For this purpose, a package element is <em>not</em> considered to
- * enclose the top-level types within that package. (A root element
+ * enclose the top-level classes and interfaces within that package. (A root element
```

Comment:
`enclose` would be better as `encloses`

Reply:
"Package element" is singular so "enclose" is proper, right?

Reply:
Ooops, mea culpa; sorry